docs/v0.5.x/pipeline-hooks.md

```diff
@@ -1,3 +1,131 @@
 ---
 title: Pipeline Hooks
 ---
+
+The build pipeline is composed of a series of hooks that each deploy plugin can optionally implement.
+
+Note:
+If multiple plugins implement the same hook they will all be called, either in alphabetical order, or
```

Comment:
Is the alphabetical order thing true? I thought it was in ember-cli's "addons" order.